docs/user/adding-glean-to-your-project.md

```diff
@@ -111,18 +111,73 @@ github "mozilla/glean" "master"
 
 </div>
 
+<div data-lang="Python" class="tab">
+
+It is recommended that you use a virtual environment for your work to isolate the dependencies for your project. There are many popular abstractions on top of virtual environments in the Python ecosystem which should help manage your project dependencies.
+
+The Python Glean bindings currently have prebuilt wheels on PyPI for x86_64 Linux only.
+
+If you're running that platform and have your virtual environment set up and activated, you can install Glean into it using:
+
+```bash
+$ python -m pip install glean_sdk
+```
+
+If you are not on x86_64 Linux, you will need to build the Glean Python bindings from source using [these instructions](../dev/python/setting-up-python-build-environment.html).
+
+</div>
+
 {{#include ../tab_footer.md}}
 
 ### Adding new metrics
 
 All metrics that your project collects must be defined in a `metrics.yaml` file.
-Add this file to your project and define it as an input file for the `sdk_generator.sh` script in the `Run Script` step defined before.
+
 The format of that file is documented [with `glean_parser`](https://mozilla.github.io/glean_parser/metrics-yaml.html).
 To learn more, see [adding new metrics](adding-new-metrics.md).
 
 > **Important**: as stated [before](adding-glean-to-your-project.md#before-using-glean), any new data collection requires documentation and data-review.
 > This is also required for any new metric automatically collected by the Glean SDK.
 
+{{#include ../tab_header.md}}
+
+<div data-lang="Swift" class="tab">
```

Comment:
In this related to the Python bindings?

Reply:
No, I just noticed reading through this that there was a note in the general section that pertained only to iOS, so I moved it to an iOS-specific section (and then added Python-specific content below it).